internal/config/node/systemd_unsupported.go

```diff
@@ -0,0 +1,32 @@
+//go:build !linux
+// +build !linux
+
+package node
+
+import (
+	"sync"
+)
+
+var (
+	systemdHasCollectModeOnce sync.Once
+	systemdHasCollectMode     bool
+	systemdHasCollectModeErr  error
+
+	systemdHasAllowedCPUsOnce sync.Once
+	systemdHasAllowedCPUs     bool
+	systemdHasAllowedCPUsErr  error
```

Comment:
I don't see a need for these if they're internal

Reply:
Removed